### ISSUE TITLE
Pimp Miner setup

### ISSUE THREAD

**Issue by Johann1982** [nicehash/NiceHashMiner-Archived#1198]
Hi, Can anyone help setting up pimp miner with nicehash multi algo switching or dual mining? When I enter the nicehash stratum on claymore dual miner for ethereum and pascal it refuses to connect. It comes back with connection closed by pool.

**Comment by commie256** [comment]
A bit offtopic: are you renting out your rigs?

**Comment by Johann1982** [comment]
Yeah, decided to rent them out with miningrigrentals. Working good so far.

**Comment by commie256** [comment]
How can we talk? Are you on SIGT discord or in skype?

**Comment by Johann1982** [comment]
johann1982hermanus@gmail.com

**Comment by commie256** [comment]
Email sent.
